Kernel/Random.h

```diff
@@ -121,10 +121,11 @@ class FortunaPRNG {
 };
 
 class KernelRng : public Lockable<FortunaPRNG<Crypto::Cipher::AESCipher, Crypto::Hash::SHA256, 256>> {
-    AK_MAKE_ETERNAL;
+    AK_INDESTRUCTIBLE;
 
 public:
     KernelRng();
+    ~KernelRng() = delete;
```

Comment:
If we give `AK_INDESTRUCTIBLE` the classname as parameter, we could make `~ClassName() = delete;` part of its definition. This would break PTYMultiplexer, but I think all the other users of this macro would benefit from it. (By which I mean: Less repetition, fewer opportunities to screw it up.)

Reply:
It would also break classes like `MemoryDevice` because you can't declare the destructor as deleted if it's a virtual destructor as far as I tried to do that yesterday.